dotnet/src/Connectors/Connectors.Anthropic/Core/AnthropicClient.cs

```diff
@@ -188,4 +399,11 @@ private void Log(LogLevel logLevel, string? message, params object[] args)
 #pragma warning restore CA2254
         }
     }
+
+    private sealed class ChatCompletionState
```

Comment:
Using as struct.

```suggestion
    private struct ChatCompletionState
```

Reply:
I prefer to leave the class here, making it struct is not much beneficial, but in the future when I implement FC this class will have more properties.